docs/book/aya/crosscompile.md

```diff
@@ -0,0 +1,21 @@
+# Cross-compiling aya-based programs
+
+The instructions below are for cross-compiling aya eBPF programs
+on Macs. The Windows instructions are on the _TODO_ list - contributions are welcome!
+
+# Cross-compiling aya-based programs on Mac
+Cross-compilation should work both on the Intel and M1/M2 Macs.
```

Comment:
Add an empty line before this line

Reply:
Cross compilation should work on both Intel and Apple Silicon Macs.

Reply:
👍 [269d421](https://github.com/aya-rs/book/pull/132/commits/269d421245837a23d5856e49bcd0259e56d5c881)